src/test/java/fr/insalyon/citi/golo/compiler/CompileAndRunTest.java

```diff
@@ -1618,4 +1610,32 @@ public void banged() throws Throwable {
     assertThat(result, not((Object)42));
 
   }
+
+  @Test
+  public void test_named_parameters() throws Throwable {
```

Comment:
https://github.com/golo-lang/golo-lang/pull/250/files#diff-d6d7e547b440d72444c214f0442a6b1aR1642

Maybe throw a `GoloCompilationException` would be cleaner but witch type of problem is this? A `PARSING` problem?